website/docs/topics/proxy-hosting.mdx

```diff
@@ -85,10 +124,43 @@ This setup is only available open-source and Enterprise customers.
 
 :::
 
-![An architecture diagram of using the proxy in a single-region, self-hosted setup.](/img/proxy-hosting-customer-single.svg)
+![An architecture diagram of using the proxy in a single-region, self-hosted setup.](/img/proxy-hosting-customer-single-edge.svg)
 
 You host everything yourself. Everything runs where and how you configure it to.
 
+**To configure Edge and the SDKs, follow steps in the [previous section on Unleash hosts the API, you host Edge](#unleash-hosts-the-api-you-host-edge)**
+
+As you might expect, doing everything yourself _is_ going to give you the most flexibility, but it's also going to be the most work. If you're already hosting Unleash yourself, though, this shouldn't be any more difficult than the previous section.
+
+As described in the [section on tradeoffs between self-hosted and Unleash-hosted setups](#unleash-hosted-or-self-hosted), running the Edge yourself gives you a number of benefits.  
+  
+
+## Multi-region
+
+![An architecture diagram of using the proxy in a multi-region, self-hosted setup.](/img/proxy-hosting-customer-multi-edge.svg)
+
+You can also use Edge for a multi-region setup. You can run Edge in a different region to the API and still connect to the API. Because Edge runs closer to your applications it still provides you benefits in terms of quicker response times. Everything should be configured as described in the [you host everything section](#you-host-everything) or the [Unleash hosts the API, you host Edge section](#unleash-hosts-the-api-you-host-edge). You can add as many Edge instances in as many extra regions as you want.
+
+  
+
+## Legacy: Unleash hosts the API, you host the Proxy
+
+:::info Recommendation
+
+This approach is no longer recommended. You should use [Unleash Edge](../reference/unleash-edge) instead of the Unleash proxy. 
```

Comment:
The migration guide does mention some limitations of Edge compared to the Proxy, it may be worth mentioning them here

Reply:
Added a note